hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestCrcCorruption.java

```diff
@@ -86,7 +86,7 @@ public void setUp() throws IOException {
    * create/write. To recover from corruption while writing, at
    * least two replicas are needed.
    */
-  @Test(timeout=50000)
+  @Test(timeout=600000)
```

Comment:
Do we really need to increase the timeout by 10x?

Reply:
Fixed, we needed to add this commit https://github.com/apache/hadoop/pull/6614/commits/01d950accaaf929542d68d784a21fee12505ab49 same as that in trunk/3.3